src/NumericalAlgorithms/DiscontinuousGalerkin/NumericalFluxes/LocalLaxFriedrichs.hpp

```diff
@@ -0,0 +1,174 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Options/Options.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/MakeWithValue.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace dg {
+namespace NumericalFluxes {
+
+/*!
+ * \ingroup DiscontinuousGalerkinGroup
+ * \ingroup NumericalFluxesGroup
+ * \brief Compute the local Lax-Friedrichs numerical flux.
+ *
+ * Let \f$U\f$ and \f$F^i(U)\f$ be the state vector of the system and its
+ * corresponding volume flux, respectively. Let \f$n_i\f$ be the unit normal to
+ * the interface. Defining \f$F_n(U) := n_i F^i(U)\f$, and denoting the
+ * corresponding projections of the numerical fluxes as \f${F_n}^*(U)\f$, the
+ * local Lax-Friedrichs flux for each variable is
+ *
+ * \f{align*}
+ * {F_n}^*(U) = \frac{F_n(U_\text{int}) + F_n(U_\text{ext})}{2} +
+ * \frac{\alpha}{2}\left( U_\text{int} - U_\text{ext}\right),
+ * \f}
+ *
+ * where "int" and "ext" stand for interior and exterior, respectively,
+ * and \f$\alpha\f$ is the maximum eigenvalue of either
+ * \f$|\Lambda(U_\text{int}; n_\text{int})|\f$ or
+ * \f$|\Lambda(U_\text{ext}; n_\text{ext})|\f$, where
+ * \f$\Lambda(U; n)\f$ is the diagonal matrix whose entries are the
+ * characteristic speeds of the system.
+ */
+template <typename System>
+struct LocalLaxFriedrichs {
+ private:
+  using char_speeds_tag = typename System::char_speeds_tag;
+  using variables_tags = typename System::variables_tags;
+
+  // The maximum characteristic speed modulus on one side of the interface.
+  struct MaxAbsCharSpeed : db::SimpleTag {
+    using type = Scalar<DataVector>;
+    static std::string name() noexcept { return "MaxAbsCharSpeed"; }
+  };
+
+ public:
+  using max_abs_speed_tag = MaxAbsCharSpeed;  // needed for testing tags.
```

Comment:
Get rid of this alias.  It is only used to check that the packaged_tags have the expected value, but it doesn't really matter what the packaged_tags are as long as the struct manages to compute the correct numerical flux from them.  If you really want to keep testing it just make the tag itself public, as that is what this line is doing anyway.

Reply:
I'll just make the tag public.